doc/news/changes/README.md

```diff
@@ -0,0 +1,40 @@
+This folder contains changelog entries
+======================================
+
+Changes between different releases in deal.II are documented in the file 
+"changes.h" that is automatically created from the files in the directories
+
+  * "major", 
+  * "minor" and 
+  * "incompatibilities".
+
+Each of these files contain a short description of a change, the authors' names
+and the date where the latter two a separated by "<br>" from the first one. 
+A typical file could look like this:
+
+    New: We can do fancy stuff now.
+    <br>
+    (John Doe, YYYY/MM/DD)
+
+and is named "YYYYMMDDJohnDoe". File names for multiple contributions from the 
```

Comment:
what if it's `JohnDoeRobertWhite...`? I think it's better to state explicitly what is actually required by the script, and what is a convenience. I suppose the only thing you really need is `YYYYMMDD` to order changes and make sure they are not conflicting. In other words,
`YYYYMMDD_this` and `YYYYMMDD_that` will also work? If not, i think the bash script should be modified. That is because all the proper authorship is already within the file
```
(John Doe, YYYY/MM/DD)
```
so how files are named shall not be important as long as you can get the date.